form_designer/email.py

```diff
@@ -55,24 +53,23 @@ def build_form_mail(form_definition, form, files=None):
     reply_to = _template_replace_list(form_definition.mail_reply_to, context_dict)
 
     mail_subject = string_template_replace(
-        (form_definition.mail_subject or form_definition.title),
-        context_dict
+        (form_definition.mail_subject or form_definition.title), context_dict
     )
 
     kwargs = {
-        'subject': mail_subject,
-        'body': message,
-        'from_email': from_email,
-        'to': mail_to,
+        "subject": mail_subject,
+        "body": message,
+        "from_email": from_email,
+        "to": mail_to,
     }
 
     if DJANGO_18:  # the reply_to kwarg is only supported in Django 1.8+ . . .
```

Comment:
This check seems awfully outdated.

It should be OK to remove the if check here and the fallback a few lines down.